python.py

```diff
@@ -47,10 +47,13 @@ def calculateScoreHelper(msg: str) -> (Optional[float], Optional[str]):
     log = r["log"]
     print(log)
     marker = "ex.py\", line "
-    first = log[log.rindex(marker) + len(marker):]
-    num_line_first = int(first[0 : find_first_index(first, '\n', ',')])
-    if filter_code(msg).strip() != v and num_line_first >= v.count("\n"):
-        return None, None
+    try:
+        first = log[log.rindex(marker) + len(marker):]
+        num_line_first = int(first[0 : find_first_index(first, '\n', ',')])
+        if filter_code(msg).strip() != v and num_line_first >= v.count("\n"):
+            return None, None
+    except:
```

Comment:
Can you be more specific with the error you're catching? and why?

Reply:
I ended up changing the score function back to what it was before hand!